Loop/AppDelegate.swift

```diff
@@ -48,6 +50,7 @@ final class AppDelegate: UIResponder, UIApplicationDelegate {
         deviceDataManager.analyticsServicesManager.application(application, didFinishLaunchingWithOptions: launchOptions)
 
         rootViewController.rootViewController.deviceManager = deviceDataManager
+        rootViewController.rootViewController.preferredGlucoseUnit = deviceDataManager.loopManager.glucoseStore.preferredUnit
```

Comment:
This seems a little strange here ... I am not sure why the VC can't grab this on its own, but I'm guessing there's a reason.  Can you please add a comment here to the code explaining this?

Reply:
This actually doesn't need to be here (though it does in the other places) - thanks for the catch!